src/recipe/ingredients.ts

```diff
@@ -49,8 +56,48 @@ function parse_ingredients(content: string): Ingredient[] {
     return recipes;
 }
 
+function parse_ingredients_recipemd(content: string): Ingredient[] {
+    const debug = false;
```

Comment:
If this is just going to be turned off here I'd rather just remove the debug logging completely here and setup a setting for it.

Reply:
I’ll take it out.